app/controllers/ChannelTestsController.scala

```diff
@@ -246,6 +246,21 @@ abstract class ChannelTestsController[T <: ChannelTest[T] : Decoder : Encoder](
 
         case Some(status) =>
           dynamoTests.updateStatuses(testNames, channel, status)
+            .flatMap(_ => {
+              /**
+               * Fetch the full data for all the updated tests and then write audits.
+               * We use .forkDaemon here to avoid delaying the response to the client
+               * and instead run this task in the background.
+               */
```

Comment:
See https://zio.dev/reference/fiber/fiber.md/#fork-in-global-scope-daemon
We want this task to run in the background, independent of the response to the client.
"So if the parent fiber terminates, the daemon fiber will not be terminated. It will only will be terminated when the global scope is closed, or its life end naturaly."